### ISSUE TITLE
[BUG] Several typescript errors in 5.0.0-beta3

### ISSUE THREAD

**Issue by mikearche** [optimizely/javascript-sdk#855]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### SDK Version

5.0.0-beta3

### Current Behavior

```
Found 68 errors in 8 files.

Errors  Files
     2  node_modules/@optimizely/optimizely-sdk/lib/core/project_config/index.ts:806
     2  node_modules/@optimizely/optimizely-sdk/lib/core/project_config/project_config_manager.ts:107
    10  node_modules/@optimizely/optimizely-sdk/lib/modules/event_processor/pendingEventsStore.ts:76 
    41  node_modules/@optimizely/optimizely-sdk/lib/optimizely/index.ts:166
     4  node_modules/@optimizely/optimizely-sdk/lib/plugins/key_value_cache/browserAsyncStorageCache.ts:27
     1  node_modules/@optimizely/optimizely-sdk/lib/utils/fns/index.ts:16
     5  node_modules/@optimizely/optimizely-sdk/lib/utils/http_request_handler/browser_request_handler.ts:42
     3  node_modules/@optimizely/optimizely-sdk/lib/utils/local_storage/tryLocalStorage.ts:28    
```

### Expected Behavior

There should be no type errors so I can run my project with typescript and optimizely successfully.

### Steps To Reproduce

1. Update to latest beta version of optimizely sdk
2. Run tsc

### SDK Type

Node

### Node Version

Node: 18.16.0
TSC: 4.7.3

### Browsers impacted

Using in NodeJS environment

### Severity

Blocking development

### Recent Change

I was hoping to get the latest version of uuid so that I can continue using optimizely without a warning being prompted. However, after updating to the beta 5.0.0 version my types stopped compiling.

**Comment by raju-opti** [comment]
Hi @mikearche, thanks for raising the issue. We are looking into it and will create internal ticket accordingly.

**Comment by raju-opti** [comment]
Hi @mikearche, we have released sdk version 5.0.0-beta4 that should solve the issue. Could you please try whether 5.0.0-beta4 works?

**Comment by mikearche** [comment]
> Hi @mikearche, we have released sdk version 5.0.0-beta4 that should solve the issue. Could you please try whether 5.0.0-beta4 works?

I tried beta4 and it seems to be fixed! I recommend having a build step in your PRs so you can't push a failing build. Happy to help.
